$MirOS: ports/mailnews/pine/patches/patch-pine_adrbklib_c,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
--- pine/adrbklib.c.orig	Thu Aug 12 19:10:47 2004
+++ pine/adrbklib.c	Wed May  4 23:28:08 2005
@@ -4916,7 +4916,7 @@ adrbk_write(ab, sort_array, enable_intr_
 	tm_now = localtime(&now);
 	for(e = deleted_aes; e; e = e->next){
 	    fprintf(ab_stream, "%s%02d/%02d/%02d#",
-		    DELETED, (tm_now->tm_year)%100, tm_now->tm_mon+1,
+		    DELETED, (int)((tm_now->tm_year)%100), tm_now->tm_mon+1,
 		    tm_now->tm_mday);
 	    if(write_single_abook_entry(e->ae, ab_stream, &this_nick_width,
 		   &this_full_width, &this_addr_width, &this_fcc_width) == EOF){
@@ -5026,7 +5026,7 @@ adrbk_write(ab, sort_array, enable_intr_
 
 	if(cnt_down != 0L)
 	  dprint(1, (debugfile,
-		"adrbk_write(%s): Can't find %d commented deleteds\n",
+		"adrbk_write(%s): Can't find %ld commented deleteds\n",
 		ab->filename ? ab->filename : "?", cnt_down));
     }
 
